README.md

```diff
@@ -140,7 +140,8 @@ private void DataReceivedNormalEvent(object sender, SerialDataReceivedEventArgs
 {
     if (e.EventType == SerialData.WatchChar)
     {
-        // We have our special character at the end of the transmission
+        // A specified character was received during the transmission and a notification event was raised.
+        // The rest of the buffer is still being received in the background, so might not be the end of transmission when reading the stream.
```

Comment:
There is only "rest of the buffer" if anything is expected after this... so this can be misleading or confusing!

For example I usually use this to detect new lines at the end of AT replies. There's the payload and the watch char at the end. Done! If anything is being received after that it will be another packet.